### PR TITLE
Feature/seext 9738 fix faulty behaviour of br tags

### PR DESCRIPTION
**Pull request by tomislav-arambasic**

### CODE REVIEW

**Review by Definitely-Not-Vlad — APPROVED** [review]
LGTM code wise, but just to make sure, this would be the expected behavior for the user? If yes, good to go, merge away at your discretion, otherwise, se if there is a way to get the expected behavior.

Hope that makes sense.

File: html/components/SimpleHtml.js [hunk 1]
```diff
@@ -71,6 +71,32 @@ class SimpleHtml extends PureComponent {
...
+   * <div>first line<br></div>
+   * <div>second line<br></div>
+   *
+   * If br is left as is, <Html> will show empty line under div, not just
+   * break into new line
+   */
+  alterChildren(node) {
+    const { children, name } = node;
+
+    if ((name === 'div' || name === 'p') && children && children.length > 1) {
```
Comment (toroz) [inline html/components/SimpleHtml.js:87]: Only for `div` & `p`?

Comment (tomislav-arambasic) [reply html/components/SimpleHtml.js:87]: Headings have top and bottom "margins" in editor & on mobile, so they're good.
Everything looks the same, on editor & mobile now.

**Comment by tomislav-arambasic** [comment]
> LGTM code wise, but just to make sure, this would be the expected behavior for the user? If yes, good to go, merge away at your discretion, otherwise, se if there is a way to get the expected behavior.
> 
> Hope that makes sense.

I wasn't sure so I checked.
When I pasted the same HTML code on 2 other HTML interpreter sites, I've seen that one `<br>`, at the end of the line, doesn't create empty line below, but only breaks the text to new line.
The other case, when there is a `div` or `p` tag, and it has no other children but `br`, then it creates a new line.

`Html` component creates empty line when it sees `<br>`.
So I handled scenario when there is other children in `div` or `p`, other than br.

Screenshot 2021-04-29 at 13 04 49

File: html/components/SimpleHtml.js (hunk 1 above)
Comment (tomislav-arambasic) [reply html/components/SimpleHtml.js:87]: Not sure if you get notified on tagging you when I edit the comment, so I'll leave this here :P 

image

**Pull request merged**